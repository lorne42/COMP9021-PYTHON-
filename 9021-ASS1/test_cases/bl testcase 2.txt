Never going to give you up.














Never going to let you down.














Never going to run around and desert you.









































Sir Roll was chilling with Sir Surprise.






Sir Surprise was chilling with Sir Astely.










Sir Surprise exclaimed, "Sir Astely is a Knight".











Sir Roll whispered, "Sir Astely is a Knave".





Who is right?