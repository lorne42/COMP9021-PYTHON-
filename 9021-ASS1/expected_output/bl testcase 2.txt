The Sirs are: Astely Roll Surprise
There are 2 solutions.